web/main/colors.css

```diff
@@ -45,6 +46,9 @@
       0 -1px oklch(0% 0 0 / 20%);
     --field-color: oklch(100% 0 0);
     --field-shadow: inset 0 1px 2px oklch(from var(--shadow-color) l c h / 8%);
+    --link-color: var(--blue-700);
+    --link-color-visited: var(--violet-700);
+    --code-inline-bg-color: oklch(from var(--neutral-200) l c h / 55%);
```

Comment:
We do not use `bg` word in other tokens. `--inline-code-color` will fit the system better.